index.html

```diff
@@ -111,18 +111,23 @@ <h2 class="subtitle">
                 <p>
                     <a href="https://deltachat.github.io/webxdc_docs/">guide book</a>
                     <br>
-                    <a href="https://github.com/deltachat/deltachat-core-rust/blob/master/draft/webxdc-dev-reference.md">reference</a>
+                    <a href="https://github.com/deltachat/deltachat-core-rust/blob/master/draft/webxdc-dev-reference.md">developer reference</a>
                 </p>
             </div>
             <div class="column is-two-thirds">
                 <h2 class="subtitle">
-                    source code for examples
+                    sample code
                 </h2>
                 <p>
-                    you can find the above examples (and more) at <a href="https://github.com/webxdc">github.com/webxdc</a> and are welcome to improve or <b>use them as a template</b> for your own, maybe far better ideas ❤️
-                    <br><br>
-                    if you prefer to start from scratch,
-                    you can clone the repo  <a href="https://github.com/webxdc/hello">github.com/webxdc/hello</a>
+                    the simplest example is our <a href="https://github.com/webxdc/hello">Hello</a>, which is useful when starting from scratch.
```

Comment:
```suggestion
                    the simplest example is <a href="https://github.com/webxdc/hello">Hello</a>, which is useful when starting from scratch.
```

if possible, "we" tend to avoid this separation into "we" and "you" ...